src/target/riscv/riscv.c

```diff
@@ -4483,12 +4467,10 @@ COMMAND_HANDLER(riscv_set_bscan_tunnel_ir)
 {
 	int ir_id = 0;
 
-	if (CMD_ARGC > 1) {
-		LOG_ERROR("Command takes at most one arguments");
+	if (CMD_ARGC > 1)
 		return ERROR_COMMAND_SYNTAX_ERROR;
-	} else if (CMD_ARGC == 1) {
+	else if (CMD_ARGC == 1)
```

Comment:
Please, fix the usage (`value` is optional):
https://github.com/riscv-collab/riscv-openocd/blob/7f054ce0cf1a1dff5b69ce4b5965331d04a8673b/src/target/riscv/riscv.c#L5302

Also consider using a plain `if` instead of an `else if` after the `return`.

Reply:
Thanks, addressed